src/Controller/ResetPasswordController.php

```diff
@@ -90,33 +89,27 @@ public function reset(Request $request, UserPasswordEncoderInterface $passwordEn
         $user = $this->resetPasswordHelper->validateTokenAndFetchUser($token);
```

Comment:
This method is missing the call to `$this->cleanSessionAfterReset()`

Reply:
Also, this line also needs to have a try-catch around it to handle failure.

Reply:
done and done.

New comments added in their respective locations regarding these 2 notes.